static/i18n/en/app.yml

```diff
@@ -254,8 +254,8 @@ receive:
       withoutDevice: Don't have your device?
     confirmAddress:
       title: Verification
-      action: Confirm address on device
-      text: To receive cryptoassets, confirm the address on your device. Click the button below to reveal your {{currencyName}} address.
+      action: Verify address on device
+      text: A {{currencyName}} receive address will be displayed on your device. Carefully verify that it matches the address on your computer.
```

Comment:
the part about "on your computer" is a bit unprecise to me. if you scan the QR code, it's not "on the computer" anymore. is there a way to say "where the address has been pasted/scanned" or something 🤔